test/extended/networking/route_advertisements.go

```diff
@@ -1505,3 +1510,17 @@ func gatherDebugInfo(oc *exutil.CLI, snifferDaemonset *v1.DaemonSet, targetNames
 		}
 	}
 }
+
+func workaroundOCPBUGS56488(oc *exutil.CLI) (bool, error) {
+	opPatch := []byte(`{"spec":{"replicas": 0}}`)
+	_, err := oc.AdminKubeClient().AppsV1().Deployments(nmstateNamespace).Patch(context.Background(), "nmstate-operator", types.MergePatchType, opPatch, metav1.PatchOptions{})
+	if err != nil {
+		return false, err
+	}
```

Comment:
maybe we need to wait nmstate-operator pods to not be there after scale them down ?